### ISSUE TITLE
New ML Kit version

### ISSUE THREAD

**Issue by teolemon** [azihsoyn/flutter_mlkit#98]
So it seems Google is putting new effort into MLKit. They have revamped the doc, and my understanding is that they released a new version.
https://android-developers.googleblog.com/2020/06/mlkit-on-device-machine-learning-solutions.html
https://developers.google.com/ml-kit/vision/barcode-scanning

**Comment by qimingweng** [comment]
Yes, it would be nice to see a Flutter MLKit implementation that doesn't require Firebase

**Comment by dlutton** [comment]
Just for fun, I created a [flutter package](https://github.com/dlutton/flutter_language_id) to use the google ml kit for language identification.  Unfortunately I don't have time to create/support a full Google ML Kit package, however if you need a one off like I created, I don't mind making it when I get some time.  Also, I'm sure Google will eventually create an official ML package, so I didn't want to waste my time with it.

**Comment by JamesLawton** [comment]
@dlutton would be great to have a one-off for the Pose API from MLkit. Anythiing  I can do to help, please let me know.

https://developers.google.com/ml-kit/vision/pose-detection

**Comment by dlutton** [comment]
@crossoveranx sounds good, I'll let you know when I get a repo started with basic functionality of that API.
